src/test/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtilsTest.java

```diff
@@ -0,0 +1,77 @@
+package io.fabric8.jenkins.openshiftsync;
+
+import io.fabric8.openshift.api.model.BuildConfig;
+import io.fabric8.openshift.api.model.BuildConfigBuilder;
+import io.fabric8.openshift.api.model.BuildConfigListBuilder;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.fabric8.openshift.client.server.mock.OpenShiftServer;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static io.fabric8.jenkins.openshiftsync.Constants.OPENSHIFT_LABELS_BUILD_CONFIG_GIT_REPOSITORY_NAME;
+
+public class OpenShiftUtilsTest {
+
+  @Rule
+  public OpenShiftServer server = new OpenShiftServer();
+
+  @Test
+  public void getBuildConfig() throws Exception {
```

Comment:
Thanks for adding the test cases :+1: 

Can we do small refactoring?
Could extract setting mock expectations logic into a separate method so test case would become easy to read and understand.